tests/base/src/test/java/org/wildfly/security/auth/client/CredentialStoreSaslAuthenticationTest.java

```diff
@@ -0,0 +1,111 @@
+package org.wildfly.security.auth.client;
```

Comment:
Each new class needs a copyright header at the top that contains the current year. You can copy/paste this from another class.

Reply:
fixed!